tests/cli/sync.test.ts

```diff
@@ -10,6 +10,7 @@ import log from '../../src/utils/logger.js';
 
 import { handler, builder } from '../../src/cli/sync.js';
 import { createAsset, updateAsset } from '../../src/assets.js';
+import { queue } from '../../src/providers/mux/provider.js';
```

Comment:
is it possible to not have to import this here and call `stopProcessingQueue`? 

I think maybe better than using an interval is to use a last requested timestamp and a set a timeout if needed.
that way every time the last request is made, the timers are cleaned up...

Reply:
Yes! I updated the implementation to use `setTimeout` instead of `setInterval`.
Now the queue is more encapsulated, there's no need to export it or use the `stopProcessingQueue`.